src/medication-request/medication-request.resource.tsx

```diff
@@ -140,7 +140,7 @@ function buildPrescriptionsTableRow(
   };
 }
 
-function computePrescriptionStatus(
+export function computePrescriptionStatus(
```

Comment:
I exported this so I could test it separately.  Not sure if that's best practice or not, but seemed low risk.

Reply:
It only really becomes an issue if someone imports it somewhere other than in a test...